source/dofs/dof_renumbering.cc

```diff
@@ -747,11 +747,10 @@ namespace DoFRenumbering
     const unsigned int n_buckets = fe_collection.n_components();
     std::vector<types::global_dof_index> shifts(n_buckets);
 
-    if (const parallel::distributed::Triangulation<dim,spacedim> *tria
-        = (dynamic_cast<const parallel::distributed::Triangulation<dim,spacedim>*>
+    if (const parallel::Triangulation<dim,spacedim> *tria
+        = (dynamic_cast<const parallel::Triangulation<dim,spacedim>*>
            (&start->get_dof_handler().get_tria())))
       {
-#ifdef DEAL_II_WITH_P4EST
```

Comment:
I think you can't just delete this -- don't you need to test for DEAL_II_WITH_MPI?